docs/10-Notes.md

```diff
@@ -79,10 +79,11 @@ Built in Chart Types
 
 There are many plugins that add additional functionality to Chart.js. Some particularly notable ones are listed here. In addition, many plugins can be found on the [Chart.js GitHub organization](https://github.com/chartjs).
 
- - <a href="https://github.com/chartjs/Chart.Zoom.js" target="_blank">Chart.Zoom.js</a> - Enable zooming and panning on charts
- - <a href="https://github.com/chartjs/Chart.Annotation.js" target="_blank">Chart.Annotation.js</a> - Draw lines and boxes on chart area
+ - <a href="https://github.com/chartjs/chartjs-plugin-zoom" target="_blank">chartjs-plugin-zoom.js</a> - Enable zooming and panning on charts
+ - <a href="https://github.com/chartjs/chartjs-plugin-annotation" target="_blank">chartjs-plugin-annotation.js</a> - Draw lines and boxes on chart area
+ - <a href="https://github.com/compwright/chartjs-plugin-draggable" target="_blank">chartjs-plugin-draggable.js</a> - Makes select chart elements draggable with the mouse
```

Comment:
Looks good, can you re-order plugins by names (starting by the chartjs-plugin-* ones):

- chartjs-plugin-annotation.js
- chartjs-plugin-deferred.js
- chartjs-plugin-draggable.js
- chartjs-plugin-zoom.js
- Chart.BarFunnel.js
- Chart.LinearGauge.js
- Chart.Smith.js